README.md

```diff
@@ -331,3 +331,29 @@ relatively long time to execute them.  Here is how you invoke the tests:
 If the tests pass then the execution will succeed and there will be no output.
 These tests may take some time to execute on Thor.  They run much faster on
 either hthor or ROXIE, due to the use of small inline datasets.
+
+### Report
+
+A report is generated based on the output of `Profile()`. The report is accessible via a Workunit's *Resources* tab within ECL Watch. For example:
+https://play.hpccsystems.com:18010/?Widget=WUDetailsWidget&Wuid=W20190430-175856#/stub/Resources-DL/Grid
+
+A report can also be viewed directly once a workunit has been created. For example:
+https://play.hpccsystems.com:18010/WsWorkunits/res/W20190430-175856/report/res/index.html
+
+Every `attribute` in the Profile result is represented by a row of information. 
+Each row of information is organized into several columns. Here is a short description 
+of each column:
+1. Type information, Cardinality Count & Filled Count
+2. Min, Avg, Max Length (for string attributes) or Mean, Std. Deviation, Quartiles (for numeric attributes)
+3. Quartile bell curve and candlestick
+    * only shown for attributes with `is_numeric` === `true`
+        * this column is omitted if the above condition fails for all attributes
+4. Cardinality Breakdown listed by count descending
+    * only shown for attributes with `cardinality_breakdown` content
+        * this column is omitted if the above condition fails for all attributes
+5. Popular Patterns 
+    * only shown for attributes with `popular_patterns` content
+        * this column is omitted if the above condition fails for all attributes
```

Comment:
@dcamper is this "short description" needed, given you have the official description elsewhere in the readme - would title just do?

Reply:
I like these descriptions.  I don't think they are redundant.

Reply:
Align this item with the previous.